modules/common/endpoint/endpoint.go

```diff
@@ -73,12 +82,14 @@ type MetalLBData struct {
 }
 
 // ExposeEndpoints - creates services, routes and returns a map of created openstack endpoint
+// NOTE: (mschuppert) deprecated, can be removed when external endpoint creation moved to openstack-operator
+// and ExposeEndpoints() can be removed
 func ExposeEndpoints(
```

Comment:
ack, we still have this here but after the whole chain is merged we can do some cleanup.